servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/BaseTestNessieApi.java

```diff
@@ -125,6 +125,7 @@
 import org.projectnessie.model.types.ImmutableGenericRepositoryConfig;
 
 /** Nessie-API tests. */
+@org.junit.jupiter.api.Tag("nessie-multi-env")
 @NessieApiVersions // all versions
```

Comment:
Are tags inherited from `@NessieApiVersions`?

Reply:
We _could_ enhance `@NessieApiVersions` as below:

```java
@Tag("nessie-multi-env")
@Target({ElementType.TYPE, ElementType.METHOD})
@Retention(RetentionPolicy.RUNTIME)
@ExtendWith(MultiVersionApiTest.class)
@Inherited
public @interface NessieApiVersions {
```

However this would work only if the test class is annotated with `@NessieApiVersions`. This is not mandatory: the annotation can also be placed on individual methods. If a test class contains only method-level occurrences of `@NessieApiVersions`, then ths test class would not be properly tagged.

Reply:
I see. I think it's fine to put `@Tag` on individual test classes.